index.js

```diff
@@ -1,13 +1,36 @@
 const express = require('express');
 const app = express();
 var bcrypt = require('bcrypt');
+var owasp = require('owasp-password-strength-test');
```

Comment:
owasp-password-strength-test
Its last commit was 2 years ago. I know it does whqt we need. But if you can find a better supported library it would be awesome.
Check fort last commit date and github starts.

Reply:
what you think about https://www.npmjs.com/package/tai-password-strength ¿?